### ISSUE TITLE
Stop Effect Automatic change color

### ISSUE THREAD

**Issue by burakcankose** [jstrausd/homebridge-simple-wled#14]
Hi,

I use the WLED plug-in fondly, but lately it turns off the effect and changes the color automatically and always makes it red. I guess Homebridge started doing it after upgrading to 1.3 versions. How can we find a solution?

[13/03/2021, 20:34:20] [WLED] Updating WLED in HomeKIT (Because of Polling) 192.168.50.218
[13/03/2021, 20:34:52] [WLED] Updating WLED in HomeKIT (Because of Polling) 192.168.50.218
[13/03/2021, 20:35:21] [WLED] Turned on 0 effect!
[13/03/2021, 20:37:23] [WLED] Set brightness to 100% 255
[13/03/2021, 20:37:23] [WLED] Changed color to 255,157,0 on host 192.168.50.218
[13/03/2021, 20:37:25] [WLED] Updating WLED in HomeKIT (Because of Polling) 192.168.50.218

**Comment by jstrausd** [comment]
Ok, I will follow the problem and see if I can find a solution for that.

**Comment by jstrausd** [comment]
Sorry for the late response, but when do it exactly change the color? How long takes it, that it changes automatically to red?

**Comment by burakcankose** [comment]
It's not a problem, sometimes it takes a few minutes, sometimes it passes in a few seconds, but on average, we can say 5 minutes.

**Comment by jstrausd** [comment]
Ok, so if I have understood correctly:
You turn on the LED and set the color for example to blue or enable an effect and after five minutes it changes automatically it's color back to red?
I have only experienced such a similar behavior, if the WLED is running multiple hours (10hours+)

**Comment by burakcankose** [comment]
yes, the whole thing happens like that, it works maximum 3-4 hours a day.  I have installed version 0.12 clean and tried the problem continues.

**Comment by dimovskidamjan** [comment]
@jstrausd I have experienced the same behaviour as @burakcankose described.

**Comment by jstrausd** [comment]
Ok, I will follow up the problem and see what it is.
But I think it could be, that the WLED-Microcontroller cannot handle that much API-Calls and then the whole controller crashes.
I will debug the Microcontroller and look to the console of the microcontroller, to see if this is an issue from the microcontroller side.

Could everyone please post his WLED-Configuration (Which microcontroller, how many LEDs, and any suspicious behaviour)?
This could help me, to find the problem/bug, Thanks.

**Comment by wsc96** [comment]
Any updates on this?

I'm running WLED 13.3 and for some reason one of the strips I have keeps randomly turning red despite being set to white.
